execution_workflows/QAPA/nextflow.config

```diff
@@ -10,10 +10,12 @@ params {
 
   // Options: Mandatory
   input                      = ''
+  gtf                        = ''
+  polyabed                   = ''
+  fasta                      = ''
```

Comment:
These get filled out at the command line call right? i.e. `nextflow main.nf --gtf <path> --polyabed <path> --fasta <path>` etc.? I don't think this is clarified in the README. If so this also means things like the 'gff' column in the sample sheet can be removed

Reply:
Thanks!